packages/manager/apps/web/client/app/hosting/database/private-sql-activation/private-sql-activation.routing.js

```diff
@@ -0,0 +1,58 @@
+import head from 'lodash/head';
+import map from 'lodash/map';
+import sortBy from 'lodash/sortBy';
+import toLower from 'lodash/toLower';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.hosting.database.private-sql-activation', {
+    url: '/private-sql-activation',
+    component: 'hostingDatabasePrivateSqlActivation',
+    redirectTo: (transition) =>
+      Promise.all([
+        transition.injector().getAsync('Hosting'),
+        transition.injector().getAsync('HostingDatabase'),
+        transition.injector().getAsync('hosting'),
+      ]).then(([Hosting, HostingDatabase, hosting]) =>
+        HostingDatabase.getPrivateDatabaseCapabilities(hosting).then(
+          (capabilities) =>
+            Hosting.getPrivateDatabasesLinked(hosting).then((privateDbs) =>
+              privateDbs.length < capabilities.length
+                ? false
+                : { state: 'app.hosting' },
+            ),
+        ),
+      ),
+    resolve: {
+      me: /* @ngInject */ (User) => User.getUser(),
```

Comment:
Isn't it already injected in the `app` state or some parent state ?

Reply:
"userInfos" is passed to child controllers the old way by using $scope, it would require some refactoring

Reply:
`user` is already injected in `app` state https://github.com/ovh/manager/blob/master/packages/manager/apps/web/client/app/app.routes.js#L11 so its inherited by this state